src/convertToObject.js

```diff
@@ -3,16 +3,29 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with
+ * styles (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS
+ * properties (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const splittedString = sourceString.split('\n');
+  const objWithStyles = {};
+
+  splittedString.forEach(string => {
+    const [prop, value] = string.split(':');
+
+    if (prop && value) {
+      objWithStyles[prop.trim()] = value.replace(/;/g, '').trim();
+    }
+  });
+
+  return objWithStyles;
```

Comment:
We are learning how to use `map`, `filter`, and `reduce` methods. Let's use them instead of `forEach`. Also, don't forget about chaining. Checklist #4